framework/src/imgui_manager.cpp

```diff
@@ -279,7 +299,7 @@ namespace gvk
 	    }
 	}
 
-	void imgui_manager::render()
+	void imgui_manager::render(unsigned int aFramePreviousRenderCallsCount)
```

Comment:
Uuh, I don't really like the inclusion of that parameter. Every `invokee` which implements `render` now has to wonder: "That must be an important information... what does it say? How can/should I use it?"

Reply:
Answering all here:

I am aware that this all could be done with semaphore consumption test call too. However I just find this solution tighter, cleaner and significantly more future proof (aside from the variable name - that needs changing, it is just a simple counter after all). 

For example: I find it weird to decide what render pass to use based on some semaphore having been consumed or not (which is currently another use case for this).

Either way, I will upload the modification tonight so we can merge this one in and if we really want to, we can discuss later.

Reply:
Changes are applied. only imageAvailable semaphore consumption test is used now.